calendar.history

```diff
@@ -501,7 +507,8 @@
 09/13	58 °C (136.4 °F) measured at el Azizia, Libya, 1922
 09/13	British defeat the French at the Plains of Abraham, just outside the walls of Quebec City, 1759
 09/13	Building of Hadrian's Wall begun, 122
-09/13	Chiang Kai-Shek becomes president of China, 1943
+09/13	Jiǎng Jièshí (Wade-Giles: Chiang3 Chieh4-shih2, known in the West as Chiang Kai-shek) becomes
```

Comment:
I think 3, 4 and 2 should be superscript and that UTF-8 supports such superscripts, isn't it? If it is, please switch those digits to superscripts.

Reply:
I learned something today. I thought you had to use HTML to get those.